src/main/java/dev/aurelium/auramobs/entities/AureliumMob.java

```diff
@@ -51,6 +51,28 @@ public AureliumMob(LivingEntity mob, int level, AuraMobs plugin) {
         Expression resHealth = new ExpressionBuilder(healthFormula).build();
         double damage = BigDecimal.valueOf(resDamage.evaluate()).setScale(2, RoundingMode.CEILING).doubleValue();
         double health = resHealth.evaluate();
+
```

Comment:
Store the result of plugin.optionString in as a variable before the if statement and reuse it, instead of calling it 3 times.